CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Changed
 
+- [#620](https://github.com/bumble-tech/appyx/pull/620) – Updated Compose to 1.5.3 & Kotlin to 1.9.10
```

Comment:
We agreed to add new stuff at the bottom

Reply:
Done